Sources/StreamChatUI/ChatChannelList/ChatChannelListSkeletonView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ChatChannelListSkeletonView.swift
+//  StreamChat
+//
+//  Created by Hugo Bernal on 21/07/22.
+//  Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatChannelListSkeletonView: _View, ThemeProvider, UITableViewDataSource {
+
+    open private(set) lazy var tableView = UITableView().withoutAutoresizingMaskConstraints
+
+    private let numberOfCells = 30
+
+    override open func setUp() {
+        super.setUp()
+
+        isUserInteractionEnabled = false
+        
+        tableView.dataSource = self
+        tableView.isScrollEnabled = false
+        tableView.register(
+            components.channelListCollectionViewSkeletonCell.self,
+            forCellReuseIdentifier: ChatChannelListCollectionViewSkeletonCell.reuseIdentifier
+        )
+    }
+    
+    override open func setUpLayout() {
+        super.setUpLayout()
+        
+        embed(tableView, insets: .init(top: 90, leading: 0, bottom: 0, trailing: 0))
```

Comment:
Why is the top spacing 90? 🤔

Reply:
This is related to the issue I mentioned you on our meeting about the skeletonView not adjusting properly considering the NavigationBar. Apparently is an issue that started being more common since iOS 15. Seems like that space of the navigationBar is now consideren on the `safeArea` and simply constraining to top won't do. So I was able to fix the issue by constraining on top to the `safeAreaLayoutGuide`.